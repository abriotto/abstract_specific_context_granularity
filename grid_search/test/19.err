  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  9%|#################                                                                                                                                                                           | 9/99 [00:00<00:01, 71.15it/s] 17%|################################1                                                                                                                                                          | 17/99 [00:00<00:01, 72.00it/s] 25%|###############################################2                                                                                                                                           | 25/99 [00:00<00:01, 72.27it/s] 34%|################################################################2                                                                                                                          | 34/99 [00:00<00:00, 75.80it/s] 43%|#################################################################################2                                                                                                         | 43/99 [00:00<00:00, 73.60it/s] 52%|################################################################################################3                                                                                          | 51/99 [00:00<00:00, 63.90it/s] 59%|#############################################################################################################5                                                                             | 58/99 [00:00<00:00, 63.78it/s] 66%|##########################################################################################################################7                                                                | 65/99 [00:00<00:00, 60.92it/s] 73%|########################################################################################################################################                                                   | 72/99 [00:01<00:00, 61.66it/s] 81%|#######################################################################################################################################################1                                   | 80/99 [00:01<00:00, 62.25it/s] 88%|####################################################################################################################################################################3                      | 87/99 [00:01<00:00, 60.12it/s] 97%|#####################################################################################################################################################################################3     | 96/99 [00:01<00:00, 66.17it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 67.24it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 36%|###################################################################6                                                                                                                        | 9/25 [00:00<00:00, 81.64it/s] 72%|######################################################################################################################################6                                                    | 18/25 [00:00<00:00, 69.35it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 66.15it/s]
