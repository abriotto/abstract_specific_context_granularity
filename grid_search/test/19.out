# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.97", "--temperature=1.5"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.713837206363678, "acc": 0.4910030663013458, "length": 2.3871617317199707, "mode": "train", "epoch": 1}
{"loss": 0.7003957629203796, "acc": 0.5073664784431458, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6980389952659607, "acc": 0.503767192363739, "length": 1.0098549127578735, "mode": "train", "epoch": 2}
{"loss": 0.69478440284729, "acc": 0.4953959882259369, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.7009686231613159, "acc": 0.4978499710559845, "length": 1.0013445615768433, "mode": "train", "epoch": 3}
{"loss": 0.6929696798324585, "acc": 0.4861878454685211, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6992219686508179, "acc": 0.4984586834907532, "length": 1.000959873199463, "mode": "train", "epoch": 4}
{"loss": 0.6953319311141968, "acc": 0.5147330164909363, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6982604265213013, "acc": 0.4920036196708679, "length": 1.0006518363952637, "mode": "train", "epoch": 5}
{"loss": 0.6927756071090698, "acc": 0.5119705200195312, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6948713064193726, "acc": 0.5116843581199646, "length": 1.0005356073379517, "mode": "train", "epoch": 6}
{"loss": 0.6927772760391235, "acc": 0.5248618721961975, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6949259638786316, "acc": 0.5132201910018921, "length": 1.0004231929779053, "mode": "train", "epoch": 7}
{"loss": 0.6928218603134155, "acc": 0.511970579624176, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6954331398010254, "acc": 0.4932350218296051, "length": 1.000404715538025, "mode": "train", "epoch": 8}
{"loss": 0.6905339956283569, "acc": 0.5267034769058228, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6953523755073547, "acc": 0.5233706831932068, "length": 1.0003583431243896, "mode": "train", "epoch": 9}
{"loss": 0.6958407163619995, "acc": 0.4972376525402069, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.6959989070892334, "acc": 0.5119914412498474, "length": 1.0002294778823853, "mode": "train", "epoch": 10}
{"loss": 0.6942977905273438, "acc": 0.5147330164909363, "length": 1.0, "mode": "test", "epoch": 10}
{"loss": 0.6953355073928833, "acc": 0.49138930439949036, "length": 1.0002856254577637, "mode": "train", "epoch": 11}
{"loss": 0.6938985586166382, "acc": 0.5082873702049255, "length": 1.0, "mode": "test", "epoch": 11}
{"loss": 0.6948588490486145, "acc": 0.5147596001625061, "length": 1.000123143196106, "mode": "train", "epoch": 12}
{"loss": 0.693847119808197, "acc": 0.531307578086853, "length": 1.0, "mode": "test", "epoch": 12}
{"loss": 0.6947324275970459, "acc": 0.5166051387786865, "length": 1.0001044273376465, "mode": "train", "epoch": 13}
{"loss": 0.6902586817741394, "acc": 0.5230203866958618, "length": 1.0, "mode": "test", "epoch": 13}
{"loss": 0.6940442323684692, "acc": 0.5070728063583374, "length": 1.0000717639923096, "mode": "train", "epoch": 14}
{"loss": 0.6945483684539795, "acc": 0.5147330164909363, "length": 1.0, "mode": "test", "epoch": 14}
{"loss": 0.6950234770774841, "acc": 0.519680380821228, "length": 1.0000756978988647, "mode": "train", "epoch": 15}
{"loss": 0.6921327114105225, "acc": 0.511970579624176, "length": 1.0, "mode": "test", "epoch": 15}
{"loss": 0.6951397061347961, "acc": 0.5098403692245483, "length": 1.000050663948059, "mode": "train", "epoch": 16}
{"loss": 0.6935328245162964, "acc": 0.511049747467041, "length": 1.0, "mode": "test", "epoch": 16}
{"loss": 0.6948800086975098, "acc": 0.510762095451355, "length": 1.0000683069229126, "mode": "train", "epoch": 17}
{"loss": 0.6959134340286255, "acc": 0.5064456462860107, "length": 1.0, "mode": "test", "epoch": 17}
{"loss": 0.6965212821960449, "acc": 0.5150676965713501, "length": 1.0000544786453247, "mode": "train", "epoch": 18}
{"loss": 0.6902095675468445, "acc": 0.533149242401123, "length": 1.0, "mode": "test", "epoch": 18}
{"loss": 0.6923526525497437, "acc": 0.5264453291893005, "length": 1.000017762184143, "mode": "train", "epoch": 19}
{"loss": 0.6978098154067993, "acc": 0.47790059447288513, "length": 1.0, "mode": "test", "epoch": 19}
{"loss": 0.6931290626525879, "acc": 0.5129152536392212, "length": 1.0000230073928833, "mode": "train", "epoch": 20}
{"loss": 0.6923784017562866, "acc": 0.5073664784431458, "length": 1.0, "mode": "test", "epoch": 20}
{"loss": 0.6936746835708618, "acc": 0.50276780128479, "length": 1.0000178813934326, "mode": "train", "epoch": 21}
{"loss": 0.6911526918411255, "acc": 0.5340700149536133, "length": 1.0, "mode": "test", "epoch": 21}
{"loss": 0.693385660648346, "acc": 0.5141404867172241, "length": 1.0002408027648926, "mode": "train", "epoch": 22}
{"loss": 0.6924923658370972, "acc": 0.5230202674865723, "length": 1.0, "mode": "test", "epoch": 22}
{"loss": 0.6931402683258057, "acc": 0.52705979347229, "length": 1.0000596046447754, "mode": "train", "epoch": 23}
{"loss": 0.6929007768630981, "acc": 0.511970579624176, "length": 1.0, "mode": "test", "epoch": 23}
{"loss": 0.6938853859901428, "acc": 0.5104494690895081, "length": 1.0002752542495728, "mode": "train", "epoch": 24}
{"loss": 0.6907107830047607, "acc": 0.5092082023620605, "length": 1.0, "mode": "test", "epoch": 24}
{"loss": 0.6927989721298218, "acc": 0.5153750777244568, "length": 1.0000039339065552, "mode": "train", "epoch": 25}
{"loss": 0.6939040422439575, "acc": 0.5128912925720215, "length": 1.0, "mode": "test", "epoch": 25}
{"loss": 0.6947511434555054, "acc": 0.5132226347923279, "length": 1.0000008344650269, "mode": "train", "epoch": 26}
{"loss": 0.6921417713165283, "acc": 0.5156537294387817, "length": 1.0, "mode": "test", "epoch": 26}
{"loss": 0.6933720707893372, "acc": 0.5215252041816711, "length": 1.0000003576278687, "mode": "train", "epoch": 27}
{"loss": 0.6961779594421387, "acc": 0.4963168203830719, "length": 1.0, "mode": "test", "epoch": 27}
{"loss": 0.6928861141204834, "acc": 0.5218332409858704, "length": 1.000009298324585, "mode": "train", "epoch": 28}
{"loss": 0.6907448172569275, "acc": 0.5165745615959167, "length": 1.0, "mode": "test", "epoch": 28}
{"loss": 0.6945482492446899, "acc": 0.5184502005577087, "length": 1.0000003576278687, "mode": "train", "epoch": 29}
{"loss": 0.6933974027633667, "acc": 0.531307578086853, "length": 1.0, "mode": "test", "epoch": 29}
{"loss": 0.6937785148620605, "acc": 0.5172201991081238, "length": 1.0000004768371582, "mode": "train", "epoch": 30}
{"loss": 0.6939849257469177, "acc": 0.5119704604148865, "length": 1.0, "mode": "test", "epoch": 30}
{"loss": 0.6925305724143982, "acc": 0.5236777067184448, "length": 1.0000022649765015, "mode": "train", "epoch": 31}
{"loss": 0.689992368221283, "acc": 0.5368324518203735, "length": 1.0, "mode": "test", "epoch": 31}
{"loss": 0.6926907300949097, "acc": 0.535977840423584, "length": 1.0000001192092896, "mode": "train", "epoch": 32}
{"loss": 0.694391667842865, "acc": 0.5027624368667603, "length": 1.0, "mode": "test", "epoch": 32}
{"loss": 0.6929254531860352, "acc": 0.5178351998329163, "length": 1.000000238418579, "mode": "train", "epoch": 33}
{"loss": 0.6907498240470886, "acc": 0.5202578902244568, "length": 1.0, "mode": "test", "epoch": 33}
{"loss": 0.6913149952888489, "acc": 0.5365928411483765, "length": 1.0, "mode": "train", "epoch": 34}
{"loss": 0.6934071779251099, "acc": 0.5147329568862915, "length": 1.0, "mode": "test", "epoch": 34}
{"loss": 0.6907895803451538, "acc": 0.5350553393363953, "length": 1.000000238418579, "mode": "train", "epoch": 35}
{"loss": 0.6918880343437195, "acc": 0.5147330164909363, "length": 1.0, "mode": "test", "epoch": 35}
{"loss": 0.6926207542419434, "acc": 0.5313653349876404, "length": 1.0000001192092896, "mode": "train", "epoch": 36}
{"loss": 0.693285346031189, "acc": 0.5248618721961975, "length": 1.0, "mode": "test", "epoch": 36}
{"loss": 0.6930385828018188, "acc": 0.5058425664901733, "length": 1.0000003576278687, "mode": "train", "epoch": 37}
{"loss": 0.6927827596664429, "acc": 0.5432780981063843, "length": 1.0, "mode": "test", "epoch": 37}
{"loss": 0.6908358931541443, "acc": 0.5242934823036194, "length": 1.0000455379486084, "mode": "train", "epoch": 38}
{"loss": 0.6915525794029236, "acc": 0.5202578902244568, "length": 1.0, "mode": "test", "epoch": 38}
{"loss": 0.6929654479026794, "acc": 0.526445209980011, "length": 1.0, "mode": "train", "epoch": 39}
{"loss": 0.6917936205863953, "acc": 0.5257827639579773, "length": 1.0, "mode": "test", "epoch": 39}
{"loss": 0.6905232667922974, "acc": 0.5202952027320862, "length": 1.000000238418579, "mode": "train", "epoch": 40}
{"loss": 0.6901590824127197, "acc": 0.520257830619812, "length": 1.0, "mode": "test", "epoch": 40}
{"loss": 0.6916313171386719, "acc": 0.5178350806236267, "length": 1.0, "mode": "train", "epoch": 41}
{"loss": 0.6929935812950134, "acc": 0.5368323922157288, "length": 1.0, "mode": "test", "epoch": 41}
{"loss": 0.6903353929519653, "acc": 0.537822961807251, "length": 1.0, "mode": "train", "epoch": 42}
{"loss": 0.6909715533256531, "acc": 0.5119705200195312, "length": 1.0, "mode": "test", "epoch": 42}
{"loss": 0.6905804872512817, "acc": 0.534747838973999, "length": 1.0, "mode": "train", "epoch": 43}
{"loss": 0.6950547099113464, "acc": 0.5101288557052612, "length": 1.0, "mode": "test", "epoch": 43}
{"loss": 0.6910733580589294, "acc": 0.5295202136039734, "length": 1.0, "mode": "train", "epoch": 44}
{"loss": 0.6937465667724609, "acc": 0.523020327091217, "length": 1.0, "mode": "test", "epoch": 44}
{"loss": 0.6893469095230103, "acc": 0.5439730286598206, "length": 1.0, "mode": "train", "epoch": 45}
{"loss": 0.6902256011962891, "acc": 0.5368324518203735, "length": 1.0, "mode": "test", "epoch": 45}
{"loss": 0.6893417835235596, "acc": 0.5464329123497009, "length": 1.0, "mode": "train", "epoch": 46}
{"loss": 0.6926641464233398, "acc": 0.5128912925720215, "length": 1.0, "mode": "test", "epoch": 46}
{"loss": 0.6889494061470032, "acc": 0.5424354076385498, "length": 1.0000001192092896, "mode": "train", "epoch": 47}
{"loss": 0.6917403936386108, "acc": 0.5257827639579773, "length": 1.0, "mode": "test", "epoch": 47}
{"loss": 0.6897285580635071, "acc": 0.5344404578208923, "length": 1.0000001192092896, "mode": "train", "epoch": 48}
{"loss": 0.6927992701530457, "acc": 0.5303868055343628, "length": 1.0, "mode": "test", "epoch": 48}
{"loss": 0.6885071396827698, "acc": 0.5427429676055908, "length": 1.000000238418579, "mode": "train", "epoch": 49}
{"loss": 0.6926413774490356, "acc": 0.521178662776947, "length": 1.0, "mode": "test", "epoch": 49}
{"loss": 0.6897253394126892, "acc": 0.5258302688598633, "length": 1.0, "mode": "train", "epoch": 50}
{"loss": 0.6952175498008728, "acc": 0.5211786031723022, "length": 1.0, "mode": "test", "epoch": 50}
{"loss": 0.6892895102500916, "acc": 0.5436652898788452, "length": 1.0, "mode": "train", "epoch": 51}
{"loss": 0.6938123106956482, "acc": 0.5064457654953003, "length": 1.0, "mode": "test", "epoch": 51}
{"loss": 0.6885083913803101, "acc": 0.5365929007530212, "length": 1.0, "mode": "train", "epoch": 52}
{"loss": 0.6925097703933716, "acc": 0.5276243090629578, "length": 1.0, "mode": "test", "epoch": 52}
{"loss": 0.689128577709198, "acc": 0.5310577750205994, "length": 1.0000001192092896, "mode": "train", "epoch": 53}
{"loss": 0.6932048201560974, "acc": 0.49355441331863403, "length": 1.0, "mode": "test", "epoch": 53}
{"loss": 0.687623143196106, "acc": 0.5439729690551758, "length": 1.000000238418579, "mode": "train", "epoch": 54}
{"loss": 0.6925021409988403, "acc": 0.5101289749145508, "length": 1.0, "mode": "test", "epoch": 54}
{"loss": 0.6886981129646301, "acc": 0.535362958908081, "length": 1.0000003576278687, "mode": "train", "epoch": 55}
{"loss": 0.6953844428062439, "acc": 0.5276243686676025, "length": 1.0, "mode": "test", "epoch": 55}
{"loss": 0.6874328255653381, "acc": 0.5353628993034363, "length": 1.0, "mode": "train", "epoch": 56}
{"loss": 0.6924728751182556, "acc": 0.5128912925720215, "length": 1.0, "mode": "test", "epoch": 56}
{"loss": 0.6882192492485046, "acc": 0.544280469417572, "length": 1.000000238418579, "mode": "train", "epoch": 57}
{"loss": 0.6938437223434448, "acc": 0.48895028233528137, "length": 1.0, "mode": "test", "epoch": 57}
{"loss": 0.6871882081031799, "acc": 0.5408978462219238, "length": 1.0, "mode": "train", "epoch": 58}
{"loss": 0.6946138739585876, "acc": 0.4797421097755432, "length": 1.0, "mode": "test", "epoch": 58}
{"loss": 0.6865277886390686, "acc": 0.5442805886268616, "length": 1.0000035762786865, "mode": "train", "epoch": 59}
{"loss": 0.6926183700561523, "acc": 0.5082873106002808, "length": 1.0, "mode": "test", "epoch": 59}
{"loss": 0.6863995790481567, "acc": 0.5387454628944397, "length": 1.0000001192092896, "mode": "train", "epoch": 60}
{"loss": 0.6937023401260376, "acc": 0.5165746212005615, "length": 1.0, "mode": "test", "epoch": 60}
{"loss": 0.6865749359130859, "acc": 0.5418205261230469, "length": 1.0, "mode": "train", "epoch": 61}
{"loss": 0.6942266821861267, "acc": 0.49447521567344666, "length": 1.0, "mode": "test", "epoch": 61}
{"loss": 0.6863053441047668, "acc": 0.5448948740959167, "length": 1.000032901763916, "mode": "train", "epoch": 62}
{"loss": 0.6952416896820068, "acc": 0.4935542941093445, "length": 1.0, "mode": "test", "epoch": 62}
{"loss": 0.6866852641105652, "acc": 0.5412054061889648, "length": 1.0, "mode": "train", "epoch": 63}
{"loss": 0.6940505504608154, "acc": 0.4972376525402069, "length": 1.0, "mode": "test", "epoch": 63}
{"loss": 0.6881929039955139, "acc": 0.5292129516601562, "length": 1.0, "mode": "train", "epoch": 64}
{"loss": 0.6948325037956238, "acc": 0.5211786031723022, "length": 1.0, "mode": "test", "epoch": 64}
{"loss": 0.6869983673095703, "acc": 0.5470479726791382, "length": 1.0, "mode": "train", "epoch": 65}
{"loss": 0.6940020322799683, "acc": 0.5018416047096252, "length": 1.0, "mode": "test", "epoch": 65}
{"loss": 0.6873746514320374, "acc": 0.535977840423584, "length": 1.000000238418579, "mode": "train", "epoch": 66}
{"loss": 0.6928913593292236, "acc": 0.5138121843338013, "length": 1.0, "mode": "test", "epoch": 66}
{"loss": 0.6861063241958618, "acc": 0.5402283072471619, "length": 1.0011647939682007, "mode": "train", "epoch": 67}
{"loss": 0.6981453895568848, "acc": 0.5073665380477905, "length": 1.0, "mode": "test", "epoch": 67}
{"loss": 0.6864186525344849, "acc": 0.5495076179504395, "length": 1.0000449419021606, "mode": "train", "epoch": 68}
{"loss": 0.6953718066215515, "acc": 0.5110496282577515, "length": 1.0, "mode": "test", "epoch": 68}
{"loss": 0.6846311092376709, "acc": 0.5476632714271545, "length": 1.000057578086853, "mode": "train", "epoch": 69}
{"loss": 0.6943783164024353, "acc": 0.520257830619812, "length": 1.0, "mode": "test", "epoch": 69}
{"loss": 0.6847765445709229, "acc": 0.5498154163360596, "length": 1.0000349283218384, "mode": "train", "epoch": 70}
{"loss": 0.6931764483451843, "acc": 0.5046040415763855, "length": 1.0, "mode": "test", "epoch": 70}
{"loss": 0.685836136341095, "acc": 0.5501232147216797, "length": 1.0000146627426147, "mode": "train", "epoch": 71}
{"loss": 0.695470929145813, "acc": 0.47605887055397034, "length": 1.0, "mode": "test", "epoch": 71}
{"loss": 0.685251772403717, "acc": 0.5405972599983215, "length": 1.0002185106277466, "mode": "train", "epoch": 72}
{"loss": 0.6936712265014648, "acc": 0.49447521567344666, "length": 1.0, "mode": "test", "epoch": 72}
{"loss": 0.6838492155075073, "acc": 0.5553505420684814, "length": 1.0000033378601074, "mode": "train", "epoch": 73}
{"loss": 0.6963045597076416, "acc": 0.5101288557052612, "length": 1.0, "mode": "test", "epoch": 73}
{"loss": 0.6838852763175964, "acc": 0.5541204810142517, "length": 1.0000017881393433, "mode": "train", "epoch": 74}
{"loss": 0.697102427482605, "acc": 0.4953960180282593, "length": 1.0, "mode": "test", "epoch": 74}
{"loss": 0.6855026483535767, "acc": 0.5467408299446106, "length": 1.0000063180923462, "mode": "train", "epoch": 75}
{"loss": 0.6956285238265991, "acc": 0.5092081427574158, "length": 1.0, "mode": "test", "epoch": 75}
{"loss": 0.6846420168876648, "acc": 0.5556581020355225, "length": 1.0000003576278687, "mode": "train", "epoch": 76}
{"loss": 0.6962169408798218, "acc": 0.49723756313323975, "length": 1.0, "mode": "test", "epoch": 76}
{"loss": 0.685017466545105, "acc": 0.537207841873169, "length": 1.000000238418579, "mode": "train", "epoch": 77}
{"loss": 0.6943755149841309, "acc": 0.528545081615448, "length": 1.0, "mode": "test", "epoch": 77}
{"loss": 0.6847313046455383, "acc": 0.5461254715919495, "length": 1.0000029802322388, "mode": "train", "epoch": 78}
{"loss": 0.6974186301231384, "acc": 0.5156538486480713, "length": 1.0, "mode": "test", "epoch": 78}
{"loss": 0.6849062442779541, "acc": 0.5516605973243713, "length": 1.000000238418579, "mode": "train", "epoch": 79}
{"loss": 0.6966032981872559, "acc": 0.5073665976524353, "length": 1.0, "mode": "test", "epoch": 79}
{"loss": 0.6848549842834473, "acc": 0.5405904650688171, "length": 1.0000007152557373, "mode": "train", "epoch": 80}
{"loss": 0.6968784332275391, "acc": 0.5147329568862915, "length": 1.0, "mode": "test", "epoch": 80}
{"loss": 0.6836230754852295, "acc": 0.5590406656265259, "length": 1.0, "mode": "train", "epoch": 81}
{"loss": 0.6949335336685181, "acc": 0.492633581161499, "length": 1.0, "mode": "test", "epoch": 81}
{"loss": 0.6843247413635254, "acc": 0.5408979058265686, "length": 1.0000027418136597, "mode": "train", "epoch": 82}
{"loss": 0.6978884935379028, "acc": 0.5174953937530518, "length": 1.0, "mode": "test", "epoch": 82}
{"loss": 0.6842727661132812, "acc": 0.554735541343689, "length": 1.000000238418579, "mode": "train", "epoch": 83}
{"loss": 0.6939388513565063, "acc": 0.5322283506393433, "length": 1.0, "mode": "test", "epoch": 83}
{"loss": 0.6836244463920593, "acc": 0.5528905391693115, "length": 1.0, "mode": "train", "epoch": 84}
{"loss": 0.6974146366119385, "acc": 0.4935542941093445, "length": 1.0, "mode": "test", "epoch": 84}
{"loss": 0.6838297843933105, "acc": 0.5525830984115601, "length": 1.0000029802322388, "mode": "train", "epoch": 85}
{"loss": 0.6985947489738464, "acc": 0.5064457058906555, "length": 1.0, "mode": "test", "epoch": 85}
{"loss": 0.6831086874008179, "acc": 0.5498154759407043, "length": 1.0, "mode": "train", "epoch": 86}
{"loss": 0.6952518224716187, "acc": 0.4852670431137085, "length": 1.0, "mode": "test", "epoch": 86}
{"loss": 0.6846400499343872, "acc": 0.547048032283783, "length": 1.000000238418579, "mode": "train", "epoch": 87}
{"loss": 0.6975460052490234, "acc": 0.5, "length": 1.0, "mode": "test", "epoch": 87}
{"loss": 0.6838893890380859, "acc": 0.5415129065513611, "length": 1.0000001192092896, "mode": "train", "epoch": 88}
{"loss": 0.6955833435058594, "acc": 0.5082873106002808, "length": 1.0, "mode": "test", "epoch": 88}
{"loss": 0.6835786700248718, "acc": 0.5390529036521912, "length": 1.000000238418579, "mode": "train", "epoch": 89}
{"loss": 0.698377251625061, "acc": 0.499079167842865, "length": 1.0, "mode": "test", "epoch": 89}
{"loss": 0.6852031350135803, "acc": 0.5461255311965942, "length": 1.000000238418579, "mode": "train", "epoch": 90}
{"loss": 0.6963736414909363, "acc": 0.511970579624176, "length": 1.0, "mode": "test", "epoch": 90}
{"loss": 0.6857839226722717, "acc": 0.5559656023979187, "length": 1.0000001192092896, "mode": "train", "epoch": 91}
{"loss": 0.6977574229240417, "acc": 0.4944751262664795, "length": 1.0, "mode": "test", "epoch": 91}
{"loss": 0.6833332777023315, "acc": 0.5547356009483337, "length": 1.0000005960464478, "mode": "train", "epoch": 92}
{"loss": 0.6955232620239258, "acc": 0.5119706392288208, "length": 1.0, "mode": "test", "epoch": 92}
{"loss": 0.6847068667411804, "acc": 0.5393601655960083, "length": 1.0000042915344238, "mode": "train", "epoch": 93}
{"loss": 0.6967718601226807, "acc": 0.492633581161499, "length": 1.0, "mode": "test", "epoch": 93}
{"loss": 0.6832331418991089, "acc": 0.5522754192352295, "length": 1.0000091791152954, "mode": "train", "epoch": 94}
{"loss": 0.7003213763237, "acc": 0.4944751262664795, "length": 1.0, "mode": "test", "epoch": 94}
{"loss": 0.6835275292396545, "acc": 0.5495080947875977, "length": 1.000000238418579, "mode": "train", "epoch": 95}
{"loss": 0.694586992263794, "acc": 0.5046040415763855, "length": 1.0, "mode": "test", "epoch": 95}
{"loss": 0.6845186352729797, "acc": 0.5531980991363525, "length": 1.0000001192092896, "mode": "train", "epoch": 96}
{"loss": 0.6964103579521179, "acc": 0.509208083152771, "length": 1.0, "mode": "test", "epoch": 96}
{"loss": 0.682695209980011, "acc": 0.553505539894104, "length": 1.0000005960464478, "mode": "train", "epoch": 97}
{"loss": 0.6956151723861694, "acc": 0.5101288557052612, "length": 1.0, "mode": "test", "epoch": 97}
{"loss": 0.6842309832572937, "acc": 0.5421279668807983, "length": 1.0, "mode": "train", "epoch": 98}
{"loss": 0.6972586512565613, "acc": 0.5147330164909363, "length": 1.0, "mode": "test", "epoch": 98}
{"loss": 0.6834115982055664, "acc": 0.5522754192352295, "length": 1.000001072883606, "mode": "train", "epoch": 99}
{"loss": 0.696872889995575, "acc": 0.5073664784431458, "length": 1.0, "mode": "test", "epoch": 99}
{"loss": 0.6833547949790955, "acc": 0.5504305958747864, "length": 1.0000016689300537, "mode": "train", "epoch": 100}
{"loss": 0.6969467997550964, "acc": 0.5110498070716858, "length": 1.0, "mode": "test", "epoch": 100}
test accuracy: 0.3239171802997589
